@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -724,6 +731,32 @@ export class TileGeometryCreator {
                     object.userData.geometryId = srcGeometry.uuid;
                 }
 
+                if (isFillTechnique(technique) && (technique.clipping || technique.transparent)) {
+                    object.onBeforeRender = (_renderer, _scene, _camera, _geometry, _material) => {
```

Comment:
this is not correct, it won't work when clipping/transparent is defined as an interpolation or a step function.
you should use `getPropertyValue(technique.clipping)` and you should re-evaluate getPropertyValue(technique.clipping/transparent) for each frame

Reply:
Great point, thanks.

Reply:
Ok, but now we have `onBeforeRender` in all `fill` object even if they have completely static and disabled clipping and/or transparency.

So this whole `onBeforeRender` installation should be guarded by variation of `if (isInterpolated(technique.clipping) || isInterpolated(technique.transparent)` ... 

Oh no, this should be even more complex
 - even non-interpolated transparency and clipping & sphere should have onBeforeRender
 - interpolated transparency or clipping should have onBeforeRender
 - else (static transparency and clipping and planar projection) statically set material attributes

That's at least how i see it from review perspective :)

Reply:
@zbigg , I will change this logic, I discussed with @ninok and we agreed to instead apply clipping to descendants of children that are currently falling back.....

Wait, that won't work, even if I clip the descendants, the children off other tiles could overlap.

Reply:
I discussed with @ninok he suggested to just enable clipping if any tile is falling back, I think this is a good compromise for now, and later we can add some special neighbour traversal code if performance is an issue. Regarding the other guards, I will take a look @zbigg , thanks.

Reply:
As discussed in standup, enabling clipping for fallback tiles would require a recompilation of the shader, so we just enable clipping for all tiles.